golang/cosmos/app/upgrade.go

```diff
@@ -92,6 +92,14 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
 			// Each CoreProposalStep runs sequentially, and can be constructed from
 			// one or more modules executing in parallel within the step.
 			CoreProposalSteps = []vm.CoreProposalStep{
+				vm.CoreProposalStepForModules(
+					// Upgrade Zoe (no new ZCF needed).
+					"@agoric/builders/scripts/vats/upgrade-zoe.js",
+				),
```

Comment:
I'm confused by the scope of this PR. Isn't `upgrade-zoe.js` already in the target branch?

Reply:
github is confused. It's showing what's supposed to be the base commit, rather than the new one. `784104` is the new commit.  Let me rattle something and see what shakes out.

Reply:
That didn't work. I'll whack it again after #10279 merges.

Reply:
rebased. It looks right now.